mobly/controllers/android_device_lib/adb.py

```diff
@@ -31,8 +31,19 @@
 DEFAULT_INSTRUMENTATION_RUNNER = 'com.android.common.support.test.runner.AndroidJUnitRunner'
 
 
-class AdbError(Exception):
-    """Raised when there is an error in adb operations."""
+class Error(Exception):
+    """Base error type for adb proxy module."""
+
+
+class AdbError(Error):
+    """Raised when an adb command encounters an error.
+
+    Args:
+        cmd: string, the adb command that timed out
```

Comment:
Should stay a list probably, in both places.

Reply:
Done